libs/designsystem/card/src/card.component.spec.ts

```diff
@@ -2,12 +2,16 @@ import { TestHelper } from '@kirbydesign/designsystem/testing';
 import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
 
 import { CardComponent } from './card.component';
+import { CardHeaderComponent } from './public_api';
+
+const OUTLINE_COLOR = 'rgb(209, 209, 209)';
```

Comment:
You can use the `DesignTokenHelper` to retreive the value via its name instead of hard-coding the value. 
```suggestion
const getColor = DesignTokenHelper.getColor;
```

Reply:
And then use further down, something like this:
`'border-block-end-color': getColor('medium')`